src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -142,6 +142,11 @@
       + "if any read operation times out, and will need to be restarted to resume "
       + "further operations.";
 
+  public static final String CONNECTION_COMPRESSION_CONFIG = "connection.compression";
+  private static final String CONNECTION_COMPRESSION_DOC = "Defines to use GZip compression on "
+          + "HTTP connection to ElasticSearch. Valid options are ``true`` and ``false``. "
+          + "Default is ``false``";
```

Comment:
/**
```suggestion
          + "Default is ``false``. The ``http.compression`` setting also needs to be set on the Elasticsearch nodes for this to work."; 
```

Reply:
@levzem or at load-balancer behind the ES node.

Reply:
Also base on https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-http.html it is enabled by default and I suggest to don't add this line.

Are you agree?

Reply:
I think it's safer to document this just in case.

Reply:
it's already done.